website/content/docs/bloop-basics.md

```diff
@@ -0,0 +1,127 @@
+JVM's [JIT](https://en.wikipedia.org/wiki/Just-in-time_compilation) and
+therefore reaches peak performance when compiling code. The difference
+between a cold and hot compiler varies. This variation is usually an order of
+magnitude large — a hot compiler can be between 10x and 20x faster than a
+cold compiler.
+
+Keeping hot compilers alive is crucial to be productive. However, it is a
+difficult task; compilers must outlive short- and long-running build
+processes, and there is no existing way to share them across different build
+tools and IDEs.
+
+Bloop is an agnostic build server that makes hot compiler accessible to all
+your toolchain — IDEs, build tools and terminals can share hot compilers and
+get peak performance at all times.
+
+By supporting popular build tools out of the box and different scenarios,
+`bloop` enforces by design best practices to be productive.
+
+### What bloop is not
+
+Bloop is **not** a new build tool, but rather a foundation block for Scala
+tooling. It can be used by Scala developers to be more productive and by
+tooling authors to write tools in a fast and reliable way.
+
+There is a lot of progress and research going into the build tools field.
+Whether you use established build tools like Bazel, Pants, sbt, Maven and
+Gradle or brand-new build tools like CBT and mill, you should benefit from
+the fastest of the workflows no matter what the internal architecture of your
+build tool is.
+
+### Components
+
+Bloop has two main components, a client and a server. If you're looking for
+ways to extend this model, head to the [Integration
+guide]({{< ref "integration-guide.md" >}}).
+
+#### `bloop`, the client
+
+The `bloop` client is the tool that you use to run any bloop command. The
+client is a python script that implements Nailgun's protocol and communicates
+with the server. It is a fast CLI tool that gives you immediate feedback from
+the server.
+
+#### `blp-server`, the server
+
+The server is called `blp-server`, and it's a long-running application that
+runs on the background and keeps the state of all your projects and compiler
+instances.
+
+When you run `bloop compile` with the client, the server receives the request
+and spawns a thread to compile your project, logging you back everything that
+the compiler outputs. This server accepts requests in parallel except for
+those tasks that are already running.
+
+
+
+### Other properties
+
+#### Runtime independent
+
+Test or run your Scala application on the JVM, Scala.js and Scala Native.
+Bloop interfaces with the API of every runtime that the Scala ecosystem
+supports so that external tools don't have to.
+
+#### Familiarity
+
+Bloop is not a long running application like the sbt shell. You run it, get
+the result, and you have your shell back. This is closer to the workflow that
+many developers love and are used to in other tools like Maven, Gradle and
+npm.
+
+#### Simplicity
```

Comment:
Vague and potentially controversial. I would drop this section or come with a more compelling argument why bloop is simple than "no DSL".